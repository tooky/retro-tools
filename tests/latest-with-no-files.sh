if retro latest
then
  echo ERROR: should have failed
fi
