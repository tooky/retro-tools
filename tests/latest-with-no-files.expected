if retro latest
then
  echo ERROR: should have failed
fi
ERROR: no retros available, try `retro add`
